openassessment/workflow/team_api.py

```diff
@@ -155,3 +163,62 @@ def get_status_counts(course_id, item_id):
         for status, count
         in counts_by_status.items()
     ]
+
+
+def cancel_workflow(team_submission_uuid, comments, cancelled_by_id):
+    """
+    Add an entry in AssessmentWorkflowCancellation table for a AssessmentWorkflow.
```

Comment:
nit: Technically it's TeamAssessmentWorkflow that should be referenced, not AssessmentWorkflow.